schedule/yast/remote_vnc/remote_vnc_controller.yaml

```diff
@@ -18,7 +18,7 @@ schedule:
   - installation/installer_timezone
   - installation/user_settings
   - installation/user_settings_root
-  - installation/installation_overview
+  - installation/installation_settings/validate_default_target
```

Comment:
no rest-api here and we are not doing anything: https://openqa.suse.de/tests/7924027#step/installation_overview/1

Reply:
Turned back.